transport-native-epoll/src/main/c/io_netty_channel_epoll_Native.c

```diff
@@ -1348,3 +1349,101 @@ JNIEXPORT jstring JNICALL Java_io_netty_channel_epoll_Native_strError(JNIEnv* en
     char* err = strerror(error);
     return (*env)->NewStringUTF(env, err);
 }
+
+JNIEXPORT jint JNICALL Java_io_netty_channel_epoll_Native_socketDomain(JNIEnv* env, jclass clazz) {
+    int fd = socket(PF_UNIX, SOCK_STREAM | SOCK_NONBLOCK, 0);
+    if (fd == -1) {
+        return -errno;
+    }
+    return fd;
+}
+
+JNIEXPORT jint JNICALL Java_io_netty_channel_epoll_Native_bindDomainSocket(JNIEnv * env, jclass clazz, jint fd, jstring socketPath) {
```

Comment:
I see:

* `JNIEnv * env`,
* `JNIEnv *env`, and
* `JNIEnv* env`

Could you use `JNIEnv* env` everywhere?